call-a-function-api-oci-clouddev.md

```diff
@@ -22,7 +20,8 @@ Key tasks include how to:
 * Deploy and test your function.
 * Create an API Gateway for your function
 * Call your function from the internet using your API Gateway.
-{% img assets/call-a-func-oracle-funcs-api-gtw-diagram.png 1200 675 "Cloud diagram" "Oracle Cloud Infrastructure Diagram showing a function executing on the service, the API gateway getting the request and returning the function, with registry in tenancy and VCN within the tenancy allowing access to the Internet" %}(https://docs.oracle.com/en-us/iaas/developer-tutorials/tutorials/functions/func-api-gtw/images/)
```

Comment:
The whole parenthetical was still there. I assume just an error, unless you were expecting it to do something?

I don't see the image that this references in the repo, did you remember to add it?

Reply:
nope, for some reason my brain thought that was proper formatting